Make.inc

```diff
@@ -207,75 +224,80 @@ ifneq (,$(findstring MSYS,$(BUILD_OS)))
 override BUILD_OS := WINNT
 endif
 
-ifeq ($(OS), WINNT)
-fPIC =
 ifeq ($(BUILD_OS), WINNT)
-PATH := $(PATH):$(build_libdir):$(build_private_libdir):/c/Program Files/7-zip
+PATH := $(PATH):$(build_libdir):$(build_private_libdir):/c/Program Files/7-zip:$(JULIAHOME)/dist-extras
```

Comment:
i saw. they are appended to the end of PATH, so there's really no reason not to try to use them (if for some reason the others don't exist)